docs/config/index.md

```diff
@@ -545,7 +545,7 @@ export default defineConfig({
 - **Type**: `boolean`
 - **Default**: `false`
 
-Allow tests and suites that are marked as only.
+Allow tests and suites that are marked as only. This also requires to set `threads` to `false`.
```

Comment:
First of all, I think you misinterpret what this flag does. 

Usually, if you have `.only` test and run it in CI, you tests will fail. This flag just allows your tests to not fail in this situation.